src/main/java/net/twitter/service/UserService.java

```diff
@@ -0,0 +1,33 @@
+package net.twitter.service;
+
+import com.auth0.json.mgmt.users.User;
+import net.twitter.dto.UserDto;
+import net.twitter.repository.UserRepository;
+
+public class UserService {
+    private static UserService instance = null;
+    private final UserRepository userRepository;
```

Comment:
don't assign the UserRepository.instance() to a property
use it directly when you want